### PR TITLE
Add support for /subpath

### PR DESCRIPTION
**Pull request by navanchauhan**

### CODE REVIEW

**Comment by navanchauhan** [comment]
The app in question has a `--webroot` flag.

Changes made in PR:

* Add support for `--webroot` flag in systemd
* Add `path_url` to `manifest.json`
* Check Process Updated
* `change_url` also changes path in `shiori.service`

**Comment by navanchauhan** [comment]
!testme

File: manifest.json [hunk 1]
```diff
@@ -34,6 +34,12 @@
                 "name": "domain",
                 "type": "domain"
             },
+            {
+                "name": "path",
+                "type": "path",
+                "example": "/example",
+                "default": "/example"
```
Comment (ericgaspar) [inline manifest.json]:
```suggestion
                "example": "/shiori",
                "default": "/shiori"
```

**Comment by navanchauhan** [comment]
@ericgaspar any reason why we shouldn't merge this directly to master?

**Comment by ericgaspar** [comment]
The workflow as stated in the contrib doc (is I recall correctly) as follow: contrib repo -> testing -> master.

**Comment by navanchauhan** [comment]
!testme

**Review by ericgaspar — APPROVED** [review]

**Pull request merged**